PixiEditor/Models/Controllers/BitmapOperationsUtility.cs

```diff
@@ -128,9 +129,10 @@ private void UseTool(List<Coordinates> mouseMoveCords, BitmapOperationTool tool,
             int thickness = sizeSetting != null ? sizeSetting.Value : 1;
 
             bool shiftDown = Keyboard.IsKeyDown(Key.LeftShift);
-
-            if (shiftDown)
+            var penTool = tool is PenTool;
+            if (shiftDown && !penTool)
             {
+                Debug.WriteLine("shift in BitmapOperationsUtility");
```

Comment:
Remove the `Debug.WriteLine`, I wouldn't do it as "if tool is not Pen", but rather "if shifDown && isShapeTool"

Reply:
wups. forgot to remove it. It should be updated now.